source/reference/mongosync.txt

```diff
@@ -161,6 +161,20 @@ source database is dropped at a later time. The ``<db>.system.profile``
 collection will not change the accuracy of user data on the
 destination.
 
+Sharded Clusters
+~~~~~~~~~~~~~~~~
+
+{+c2c-product-name+} supports replication between sharded clusters.
+Individual shards are replicated in parallel from the source cluster to
+the destination cluster, however a :ref:`chunk migration
+<sharding-chunk-migration>` or similar source update could move
+documents to a new source shard during replication.
+
+To preserve event order on the destination cluster,
+{+c2c-product-name+} automatically adjusts the :ref:`change events
```

Comment:
Not quite sure what "adjusts" means here. But I assume this is fine. This is implementation details of how we handle out of order events while maintaining eventual consistency. I guess the end users don't have to know. So this is probably fine then.

Just want to provide more context here: https://jira.mongodb.org/browse/REP-33 is the project to implement the core replication machinery for sharded clusters. I don't think there is much we need to document here other than a statement that we support sharded clusters. So I think the level of details we have here is probably fine.

Reply:
Acknowledged